### DISCUSSION TITLE
Discussion of principles. Stay to the roots.

### DISCUSSION THREAD

**Discussion by user4andre** [FossifyOrg/General-Discussion#623] [General]
### Checklist

- [X] I made sure that there are **no existing issues** - [open](https://github.com/FossifyOrg/Keyboard/issues) or [closed](https://github.com/FossifyOrg/Keyboard/issues?q=is%3Aissue+is%3Aclosed) - to which I could contribute my information.
- [X] I made sure that there are **no existing discussions** - [open](https://github.com/FossifyOrg/Keyboard/discussions) or [closed](https://github.com/FossifyOrg/Keyboard/discussions?discussions_q=is%3Aclosed) - to which I could contribute my information.
- [X] I have read the FAQs inside the app (Menu -> About -> FAQs) and my problem isn't listed.
- [X] **I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.**
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/FossifyOrg/Keyboard/blob/main/CONTRIBUTING.md).
- [ ] I optionally [donated](https://fossify.org/donate) to support the Fossify mission.

### Feature description

I want a keyboard app that is as simple as possible.

### Why do you want this feature?

If all the desired customizations are implemented, Fossify Keyboard is the wrong choice for me!

Or divide the development into 2 different versions. A basic version and one with extras.

**Comment by Aga-C** [comment]
1. Feature requests and enhancements with `needs triage` aren't decided if they will be developed or not.
2. Even if we add some bigger features like suggestions, it will be possible to turn them off in Settings.
